### PR TITLE
client: update pd/client to repair kvproto compatibility

### PR DESCRIPTION
**Pull request by HuSharp**
Signed-off-by: husharp <jinhao.hu@pingcap.com>

as https://github.com/tikv/pd/pull/5861 described, will merged after 5861 to make sure compatibility.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,13 +1,388 @@
...
+cloud.google.com/go v0.93.3/go.mod h1:8utlLll2EF5XMAV15woO4lSbWQlk8rer9aLOfLh7+YI=
+cloud.google.com/go v0.94.1/go.mod h1:qAlAugsXlC+JWO+Bke5vCtc9ONxjQT3drlTTnAplMW4=
+cloud.google.com/go v0.97.0/go.mod h1:GF7l59pYBVlXQIBLx3a761cZ41F9bBH3JUlihCt2Udc=
+cloud.google.com/go v0.99.0/go.mod h1:w0Xx2nLzqWJPuozYQX+hFfCSI8WioryfRDzkoI/Y2ZA=
+cloud.google.com/go v0.100.1/go.mod h1:fs4QogzfH5n2pBXBP9vRiU+eCny7lD2vmFZy79Iuw1U=
+cloud.google.com/go v0.100.2/go.mod h1:4Xra9TjzAeYHrl5+oeLlzbM2k3mjVhZh4UqTZ//w99A=
+cloud.google.com/go v0.102.0/go.mod h1:oWcCzKlqJ5zgHQt9YsaeTY9KzIvjyy0ArmiBUgpQ+nc=
+cloud.google.com/go v0.102.1/go.mod h1:XZ77E9qnTEnrgEOvr4xzfdX5TRo7fB4T2F4O6+34hIU=
+cloud.google.com/go v0.104.0/go.mod h1:OO6xxXdJyvuJPcEPBLN9BJPD+jep5G1+2U5B5gkRYtA=
+cloud.google.com/go v0.105.0/go.mod h1:PrLgOJNe5nfE9UMxKxgXj4mD3voiP+YQ6gdt6KMFOKM=
```
Comment (nolouch) [inline go.sum:34]: why so many versions?

Comment (HuSharp) [reply go.sum:34]: generate by `go mod tidy` after updating `pd/client` to the master version 🤔

**Review by disksing — APPROVED** [review]

**Review by rleungx — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**